tests/integration/store/rollback_test.go

```diff
@@ -4,26 +4,63 @@ import (
 	"fmt"
 	"testing"
 
-	"cosmossdk.io/simapp"
```

Comment:
This was using simapp. We cannot move that in store/ because of it needs an app and store cannot depend on the SDK. I think this test actually fits better in e2e then.